build/helm/edgemesh/charts/agent/templates/04-configmap.yaml

```diff
@@ -65,3 +65,6 @@ data:
       tunnel:
         enable: {{ .Values.modules.tunnel.enable }}
         listenPort: {{ .Values.modules.tunnel.listenPort }}
+        security:
+          enable: {{ .Values.modules.tunnel.security.enable }}
+          httpServer: {{ .Values.modules.tunnel.security.httpServer }}
```

Comment:
why not also add tlsCAFile, tlsPrivateKeyFile, tlsCertFile?

Reply:
fix